content/docs/02.installation/08.aws-ec2.md

```diff
@@ -22,11 +22,11 @@ First, create an EC2 instance. To do so, [go to the AWS console and choose EC2](
 4. Create a key-pair to securely connect to your instance. This key is needed to connect through SSH in the following steps.
 5. Create a security group that allows SSH traffic from your IP. Also allow HTTPS traffic.
 
-![ec2 creation](/docs/administrator-guide/deployment/aws-ec2/ec2_setup1.png)
+![ec2 creation](../../../public/docs/administrator-guide/deployment/aws-ec2/ec2_setup1.png)
 
-![ec2 key pair](/docs/administrator-guide/deployment/aws-ec2/ec2_setup2.png)
+![ec2 key pair](../../../public/docs/administrator-guide/deployment/aws-ec2/ec2_setup2.png)
 
-![ec2 network subgroup](/docs/administrator-guide/deployment/aws-ec2/ec2_setup3.png)
+![ec2 network subgroup](../../../public/docs/administrator-guide/deployment/aws-ec2/ec2_setup3.png)
 
```

Comment:
you can change the images and keep the same path

the ../../../ are not needed and lead to those images not rendering 

![image](https://github.com/kestra-io/docs/assets/86264395/7feb7a3e-c8c8-4292-9338-0dc88da5b903)